### ISSUE TITLE
Image preprocessing statistics for dataset

### ISSUE THREAD

**Issue by jlindsey15** [google-research/simclr#12]
Hello,

Thanks so much for making this code available!  I am trying to use the pretrained SimCLR models to extract features from my own custom dataset.  I have done this successfully with other models trained using standard methods ("successfully" meaning that the extracted features are effective at classifying my custom dataset).  With SimCLR however the extracted features seem to be only slightly better than random.  I am wondering if this is because the input statistics to SimCLR are different from those in most ImageNet-trained models.  I normalize my data to the Imagenet mean/std of mean=[0.485, 0.456, 0.406], std=[0.229, 0.224, 0.225].  Should I be using different normalization for SimCLR?

**Comment by cgarciae** [comment]
I think here you have normalize like this:

```python
image = image / 255.0
```

**Comment by chentingpc** [comment]
yes, the preprocessing code takes and produces a [0, 1] ranged input.

**Comment by ClaraBing** [comment]
For CIFAR 10, my understanding is that the pre-processing parameters provided in README do not contain noise, blur, cutout, rotate, or Sobel filtering.

Did I misunderstand the setting, or were these transformation left out because they are less essential? Thank you!

**Comment by chentingpc** [comment]
For results (besides the augmentation ablation results in 3.1) in the paper, CIFAR or ImageNet, we only use the following augmentations: random crop (+random flip and resize), color distortion and blur. Other augmentations are not added as we did not systematically explore a more complicated augmentation strategy, and preliminary ablations (in 3.1) show crop and color are the most important ones.

**Comment by ClaraBing** [comment]
I see, thank you very much!
